crates/runtime/src/embeddings/vector_search.rs

```diff
@@ -40,8 +40,8 @@ use snafu::prelude::*;
 
 #[derive(Debug, Snafu)]
 pub enum Error {
-    #[snafu(display("Data source {} does not exist", data_source))]
```

Comment:
This is just moving datasets from `String` -> `TableReference`, where appropriate.